src/runtime/composables/useApiData.ts

```diff
@@ -1,9 +1,9 @@
 import { computed, reactive } from 'vue'
 import { hash } from 'ohash'
-import type { FetchError } from 'ofetch'
+import type { IFetchError } from 'ofetch'
```

Comment:
This is technically correct, although I think using `FetchError` is simpler, since then users can check `instanceof FetchError` etc.
A fix in ofetch landed to augment the `FetchError` to include all `IFetchError` properties: https://github.com/unjs/ofetch/commit/26fd44bd459d6c384f477d86795cdf09aa3843c4

We're safe to use `FetchError` (as we did in ofetch < 1.2.0, there were recent changes which broke the `FetchError` usage as type).

Reply:
I just checked, and `data` does not have a prototype. It is a plain object. `error.data isinstance FetchError` seems to always return `false` and `Object.getPrototypeOf(error.data)` returns `{}`.